big_tests/tests/mam_SUITE.erl

```diff
@@ -538,7 +542,8 @@ prefs_cases() ->
      run_set_and_get_prefs_cases].
 
 impl_specific() ->
-    [check_user_exist].
+    [check_user_exist,
+     pm_failed_to_decode_message_in_database].
```

Comment:
Why in this group and not in some regular pm one? Just asking 🤔

Reply:
cannot be run in parallel. because it changes module config mid test.